user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Data/Spectrum_Preset_Data.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: Spectrum_Preset_Data
+---
+
+# Spectrum preset data
+
+The *Spectrum presets* data category allows you to use data from [spectrum presets](xref:Using_Spectrum_Analysis_presets) in your DataMiner System as input for components in your dashboards or low-code apps.
+
+When you expand the *Spectrum presets* section in the *Data* pane, you first need to select a Spectrum Analyzer protocol. Once selected, all available spectrum presets are listed. To quickly find a specific spectrum preset, use the filter box at the top of the data section.
```

Comment:
You can also use the section/category as data and filter it using an element reference.